README.md

```diff
@@ -42,60 +42,69 @@ c.GitPlus.github_token = '<your-github-access-token>'
 
 After installation, start JupyterLab normally & you should see "Git-Plus" as a new menu item at the top.
 
+#### 2.1 Configuring a custom reviewnb URL
```

Comment:
#### 3. [OPTIONAL] Configuration for Self Hosted ReviewNB


##### 3.1 Configuring Self Hosted ReviewNB URL
.......


(in the future)
##### 3.2 Configuring Self Hosted GitHub Enterprise Server URL
.......

Reply:
Done